flake.nix

```diff
@@ -206,6 +206,7 @@
                 # ormolu
                 # stylish-haskell
                 opentelemetry-extra
+                haskell-language-server
```

Comment:
I think `haskell-language-server` here is not the one in nixpkgs, but the one being hacked. So after each modification, entering the shell would rebuild HLS.

Reply:
Oh you're right, my bad 🤦‍♂️